### ISSUE TITLE
When using voiceover unable to access expanded list of project combo box: A11y_.NET maui_user can creat a tak_Screen reader

### ISSUE THREAD

**Issue by kapilvaishna** [dotnet/maui#30897]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yTCS; #A11yMAS; #WCAG1.3.2; #ScreenReader; #BM_.NETMaui_Mac_JUL2025; #DesktopApp; #Device; #FTP; #A11ySev1; #.NET MAUI SDKs; #Mac; #VoiceOver;

## Environment Details:
Application: Developer Balance (.NET Maui)
MAC
M2

## Repro Steps:

1. Turn on Voiceover
2. Install and open the Developer Balance app
3. Press Control + Option + Right arrow key to navigate "Add task" button and press  Control + Option + space key.
4. Press Control + Option + Right arrow key to navigate Project combo box and press  Control + Option + space key.
5. Observe the issue 

## Actual Result:
When using voiceover and pressing  Control + Option + space key to expand list of project combo box than unable to access expanded list.

**Note:**
Same issue repro with "Category" combo box present in create a project page

## Expected Result:
When list of project combo box expand, focus should automatically shift to expanded list. 

## User Impact:
Screen reader user will unable to access the expanded list of project combo box.

## Attachment

https://github.com/user-attachments/assets/ed72a7ad-cc12-4739-8e0d-93d245ca96ac

### Similar issue

https://github.com/user-attachments/assets/26980f0c-3a98-411b-8a51-e33cf34c8d8a

**Comment by ghost** [comment]
This issue has been verified in Visual Studio Code 1.102.2 with MAUI versions 9.0.0 and 9.0.90. It can be reproduced on MacCatalyst platform.

**Comment by SuthiYuvaraj** [comment]
Hi @kapilvaishna  , I am unable to reproduce the issue on my Mac machine. For your reference, I have included the Mac system details along with an output video demonstrating the behavior.

**Mac Details:**
Chip: Apple M3 Max
VS Code version: 1.103.1  

**Output video:**
<video width="70" height="150" alt="Before Fix" src="https://github.com/user-attachments/assets/4903bf9a-a589-45e8-a82d-e9ac4942134e">

**Comment by SuthiYuvaraj** [comment]
@kapilvaishna, I have re-verified this on my Mac machine and was unable to reproduce the issue. Could you kindly check on your end using a similar Mac setup to see if the behavior persists?

**Comment by kapilvaishna** [comment]
Issue is reproducing for me in below environment

@SuthiYuvaraj Could you please confirm the Developer balance version also where issue is not reproducing for you. if version same than I will try  to check on M3. 

Developer balance version 10 (10)
Chip Apple M2

https://github.com/user-attachments/assets/5b66659e-6b75-42b5-9c69-ce88c39a495e

**Comment by kapilvaishna** [comment]
@PureWeen issue is still reproducing in Version 11

https://github.com/user-attachments/assets/e2c6ad33-608b-4fac-986d-1b185b2f458a

**Comment by SuthiYuvaraj** [comment]
Hi @kapilvaishna  ,I am unable to reproduce the issue on my Mac machine using .NET 11. Please refer to the video below for reference.

https://github.com/user-attachments/assets/49e9a945-7279-4353-86c3-1de739ef19d0
